commons/src/main/kotlin/com/simplemobiletools/commons/extensions/Context-storage.kt

```diff
@@ -155,12 +155,24 @@ fun Context.isPathOnInternalStorage(path: String) = internalStoragePath.isNotEmp
 
 private const val ANDROID_DATA_DIR = "/Android/data/"
 private const val ANDROID_OBB_DIR = "/Android/obb/"
+private const val DOWNLOAD_DIR = "Download"
```

Comment:
this needs more research, at renaming it is both Download and Documents + their subfolders behaving differently, not just Download itself

Reply:
On investigating, we can select tree URI of all directories except
-  the root of Internal storage,
-  the root of SD card and, 
- the Download folder of either the Internal Storage or SD card

The information is [here](https://developer.android.com/training/data-storage/shared/documents-files#document-tree-access-restrictions)